<?xml version="1.0"?>
<!--
<author>Swapneel Naphade</author>
<email>snaphade@umd.edu</email>
<version>1.0</version>
-->

<launch>

	<!-- Launch RVIZ -->
	<node type="rviz" name="rviz" pkg="rviz" args="-d $(find auto_drone)/rviz/gatedetector_test.rviz" />

	<!-- Launch gate_detector node -->
	<node pkg="auto_drone" type="gate_detection_and_estimation.py" name="gate_detector" output="screen"/>
	

</launch>
